istio.deps

```diff
@@ -4,6 +4,6 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "4d8eb98dee59b8cd57962230c0430fa2a486c05e"
+		"lastStableSHA": "255ac3c8181844e01b20c14d91147a9ae2a11b59"
```

Comment:
Should be 9b91ac6ec32483194717d5486ae34d0997b70692 instead.
See https://github.com/istio/proxy/commits/release-1.1